src/main/java/practice/StreamPractice.java

```diff
@@ -75,6 +100,11 @@ public List<String> getCatsNames(List<Person> peopleList, int femaleAge) {
      * parametrized with Candidate in CandidateValidator.
      */
     public List<String> validateCandidates(List<Candidate> candidates) {
-        return Collections.emptyList();
+        CandidateValidator validator = new CandidateValidator();
+        return candidates.stream()
+                .filter(validator)
```

Comment:
```suggestion
        return candidates.stream()
                .filter(new CandidateValidator())
```
code would be much cleaner